variables.tf

```
@@ -27,6 +27,12 @@ variable "dead_letter_target_arn" {
   description = "The ARN of an SNS topic or SQS queue to notify when an invocation fails"
 }
 
+variable "enable_all_egress_rule" {
```

```suggestion
variable "create_ allow_all_egress_rule" {
```

I think `create` is more descriptive than `enable` 🙂

---

updated the var name 👍